Generate.py

```diff
@@ -182,20 +182,35 @@ def main(args=None, callback=ERmain):
          for fname, yamls in weights_cache.items()}
 
     if meta_weights:
+        # resolve triggers in each game section if they exist
+        for category in meta_weights.values():
+            if "triggers" in category:
+                meta_weights = roll_triggers(meta_weights, category["triggers"])
+        # each game "section"
         for category_name, category_dict in meta_weights.items():
             for key in category_dict:
+                # rolling triggers adds the generator version to the weights. not sure why but didn't want to break it
+                if key == "triggers" and category_name == "_Generator_Version":
```

Comment:
ah no i actually changed this locally already. the reason it isn't working as expected is because the trigger function expects certain options to be root in the yaml, but those options are in the `none` category instead in the meta file. `game` is the one in particular that is breaking. so i can do ```yaml
game: Hollow Knight
triggers:
  trigger_off_hollow_knight``` but if i do ```yaml
null:
  game: Hollow Knight
  triggers:
    trigger_off_hollow_knight``` which is the preferred format for the meta file it doesn't work. solution here is to either change the meta yaml so it expects the same format as the normal player yamls - which is doable - or change how the trigger functioning happens for the meta. i'm partially leaning towards the former solution atm.